.changeset/slimy-walls-knock.md

```diff
@@ -0,0 +1,5 @@
+---
+"@vue-storefront/sdk": patch
+---
+
+[FIXED] BaseConfix extensibility. Now, it allows to add additional custom properties.
```

Comment:
Typo in `baseconfix` and what kind of custom properties?

Reply:
<3